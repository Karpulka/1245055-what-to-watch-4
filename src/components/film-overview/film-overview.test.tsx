import * as React from "react";
import renderer from "react-test-renderer";
import FilmOverview from "./film-overview";

const overview = {
  description: `This is Description. TCHK.`,
  rating: 8.5,
  voiceCount: 214,
  director: `And he's Director`,
  actorList: [`Actor`, `Actor`, `Actor`]
};

it(`Render FilmOverview`, () => {
  const tree = renderer
    .create(<FilmOverview overview={overview} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
