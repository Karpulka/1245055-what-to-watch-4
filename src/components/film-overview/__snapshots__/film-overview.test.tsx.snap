// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render FilmOverview 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      8.5
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
        214
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    This is Description. TCHK.
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        And he's Director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Actor, Actor, Actor
         and other
      </strong>
    </p>
  </div>,
]
`;
